liquibase-standard/src/main/java/liquibase/snapshot/jvm/ColumnSnapshotGenerator.java

```diff
@@ -477,9 +477,9 @@ protected DataType readDataType(CachedRow columnMetadataResultSet, Column column
          */
         int jdbcType = columnMetadataResultSet.getInt("DATA_TYPE");
 
-        // Java 8 compatibility notes: When upgrading this project to JDK8 and beyond, also execute this if-branch
-        // if jdbcType is TIMESTAMP_WITH_TIMEZONE (does not exist yet in JDK7)
-        if (jdbcType == Types.TIMESTAMP) {
+        if (jdbcType == Types.TIMESTAMP || jdbcType == Types.TIMESTAMP_WITH_TIMEZONE
```

Comment:
This `|| jdbcType == Types.TIMESTAMP_WITH_TIMEZONE` wasn't here before.. is it required for asany too?

Reply:
Once SAP fixes the bug (i. e. reports a TIMESTAMP WITH TIMEZONE as a TIMESTAMP_WITH_TIMEZONE) then it will be necessary. Until then (i. e. as long as SAP reports a TIMESTAMP WITH TIMEZONE as a VARCHAR) it is not essentially needed to resolve *this* issue. But see, it was needed since long time, see the removed comment, so I thought we can fix both issues in one PR.